src/feed/subscription/RealtimeSubscription.ts

```diff
@@ -22,17 +22,20 @@ class RealtimeSubscription extends Subscription {
     }
 
     async _startSubscribe(tickHistoryRequest: TCreateTickHistoryParams) {
+        const contract_info = this.contractInfo as ProposalOpenContract;
         //here we include duration = 'ticks' && exclude duration = 'seconds' which hasn't tick_stream, all_ticks, tick_count (consist of 15-86.400 ticks)
-        if(!this.shouldFetchTickHistory && !!(this.contractInfo as ProposalOpenContract).tick_stream){
+        if(!this.shouldFetchTickHistory && !!contract_info.tick_stream){
             const [tickHistoryPromise, processTickHistory] = this._getProcessTickHistoryClosure();
```

Comment:
@akmal-deriv let's place it outside after line 25 because it's done both inside if and else🙂